packages/web/src/components/ui/Card/CardChips.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import CardCommon, { CardCommonProps } from './CardCommon';
+import styles from './Card.module.scss';
+import Chip from '../Chip/Chip';
+import ChipGroup from '../ChipGroup/ChipGroup';
+
+export interface CardChipsProps extends CardCommonProps {
+  chips: string[];
+  restPackages?: number;
+}
+
+const CardChips = ({ title, icon, chips, restPackages }: CardChipsProps) => {
```

Comment:
Looks like this component needs a more explicit name, along the lines of `PackageUsageByHostnameCard`